src/js/settings-list.js

```diff
@@ -429,6 +429,12 @@ module.exports = [
         }
     },
     {
+        name: 'Hide Last Broadcast Banner',
```

Comment:
instead of making a setting in our menu, why not improve upon twitch's feature by adding a close button to their banner that remembers the setting